vspace/src/main/webapp/WEB-INF/views/layouts/navbar.html

```diff
@@ -44,6 +70,11 @@
 		        </div>
 		      </div>
 		    </nav>
+		    <div id="aboutContent" th:if="${aboutPageTitle!=null}">
+                        <span class="icon-circle-close closebtn" th:onclick="closeExhibitionDescription()"></span>
+                        <div class="aboutDescription">[[${aboutPageTitle}]]</div>
+                        <div th:if="aboutPageText != null &&!#strings.isEmpty(aboutPageText)} " class="aboutDescription">[[${aboutPageText}]]</div>
```

Comment:
The page doesn't load on staging. My guess is that there is an opening `{` missing here.

Reply:
Yes, have fixed this now